stl/inc/ranges

```diff
@@ -411,31 +421,44 @@ namespace ranges {
     };
 
     // clang-format off
-    template <_Copy_constructible_object _Ty>
-        requires (copyable<_Ty>
-            || (is_nothrow_copy_constructible_v<_Ty>
-                && (movable<_Ty> || is_nothrow_move_constructible_v<_Ty>)))
-    class _Copyable_box<_Ty> { // provide the same API more efficiently when we can avoid the disengaged state
+    template <class _Ty>
+    concept _Use_simple_movable_box_wrapper =
+        // 1. If copy_constructible<T> is true, movable-box<T> should store only a T if either T models copyable, or
+        // is_nothrow_move_constructible_v<T> && is_nothrow_copy_constructible_v<T> is true.
+        (copy_constructible<_Ty> && (
+            copyable<_Ty>
+            || (is_nothrow_move_constructible_v<_Ty> && is_nothrow_copy_constructible_v<_Ty>)
+        ))
+        // Otherwise, movable-box<T> should store only a T if either T models movable or

```

Comment:
This started with `1.` from the Standardese, but the "Otherwise" is missing `2.` Either both numbers should be quoted, or they both should be omitted.